CHANGELOG.md

```diff
@@ -28,6 +28,13 @@
   E.g. the associated metrics for vhost-user block device with endpoint
   `"/drives/rootfs"` will be available under `"vhost_user_block_rootfs"`
   in the metrics json object.
+- [#4138](https://github.com/firecracker-microvm/firecracker/pull/4138),
+  [#4170](https://github.com/firecracker-microvm/firecracker/pull/4170),
+  [#4223](https://github.com/firecracker-microvm/firecracker/pull/4223),
+  [#4226](https://github.com/firecracker-microvm/firecracker/pull/4226):
```

Comment:
we already have an entry for 4226, is this intentional to mention all changes together?

Reply:
hm, indeed, looks like we mention it twice. Would it make sense to remove an entry about vhost-user-blk metrics and assume it a part of the implementation?